DuckDuckGoTests/MockDependencyProvider.swift

```diff
@@ -46,6 +46,7 @@ class MockDependencyProvider: DependencyProvider {
     var networkProtectionKeychainTokenStore: NetworkProtectionKeychainTokenStore
     var networkProtectionTunnelController: NetworkProtectionTunnelController
     var connectionObserver: NetworkProtection.ConnectionStatusObserver
+    var serverInfoObserver: NetworkProtection.ConnectionServerInfoObserver
```

Comment:
We need to keep an instance of this alive from the very beginning so that it can listen for connection changes. Otherwise, when we enter the VPN UI, we won't have any server info to display.